examples/kubernetes-mount-config-map-volumes/README.md

```diff
@@ -0,0 +1,184 @@
+## Mount config map volumes to deployment 
+
+- This sample runs simple ballerina hello world service with config map mounts.
+- K8S config maps are intended to hold config information.
+- Putting this information in a config map is safer and more flexible than putting it verbatim in a pod definition or in a docker image.
+
+### How to run:
+
+1. Compile the ballerina module. Command to deploy kubernetes artifacts will be printed on build success.
+```bash
+$> bal build 
+Compiling source
+	hello/hello:0.0.1
+
+Creating balas
+	target/bala/hello-2020r2-any-0.0.1.bala
+
+Running Tests
+
+	hello/hello:0.0.1
+	No tests found
+
+
+Generating executables
+	target/bin/hello.jar
+
+Generating artifacts...
+
+	@kubernetes:Service 		    - complete 1/1
+	@kubernetes:Secret 			    - complete 1/1
+	@kubernetes:ConfigMap 			- complete 2/2
+	@kubernetes:Deployment 			- complete 1/1
+	@kubernetes:HPA 			 - complete 1/1
+	@kubernetes:Docker 			 - complete 2/2
+
+	Execute the below command to deploy the Kubernetes artifacts:
+	kubectl apply -f /Users/anuruddha/workspace/ballerinax/module-ballerina-c2c/samples/sample5/target/kubernetes/hello
+
+	Execute the below command to access service via NodePort:
+	kubectl expose deployment hello-hello-0-0-deployment --type=NodePort --name=hello-hello-0-0-svc-local
+```
+### How to write:
+This segment shows how a c2c segment is mapped into cloud element. 
+
+1. `Cloud.toml` segment
+   ```toml
+      [[cloud.config.maps]]
+		file="./conf/data.txt" # Path of the external file
+		mount_path="/home/ballerina/data" # Path of the file in the container 
+
+		[[cloud.config.files]]
+		file="./conf/Config.toml" # Path of the external file
+		name="sample5-config-map" 
+
+   ```
+2. Kubernetes YAML file segment
+   ```yaml
+       apiVersion: "v1"
+		kind: "ConfigMap"
```

Comment:
Formatting seems wrong when displayed in markdown. 
https://github.com/Thevakumar-Luheerathan/module-ballerina-c2c/tree/restructure-samples/examples/kubernetes-mount-config-map-volumes

Shall we recheck in all the yamls inside markdowns?